src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/ElasticSearch.php

```diff
@@ -85,4 +85,34 @@ public function getVersion(): string
 
         return $this->version;
     }
+
+    /**
+     * @return array
```

Comment:
Add description to the method, and what settings are expected to be retrieved

Reply:
Added